### PR TITLE
epic: drop IE11 support and legacy properties (`main-next` -> `main-beta`)

### PR DESCRIPTION
**Pull request by ala-n**
Closes: #1462
Closes: #1461

### CODE REVIEW

File: src/modules/esl-utils/misc/array.ts [hunk 1]
```diff
@@ -36,14 +38,8 @@ export function unwrap(value: any): any {
...
-export const uniq = <T> (arr: T[]): T[] => {
-  if (arr.length < 2) return arr.slice(0);
-  const result: T[] = [];
-  const set = new Set<T>();
-  arr.forEach((item) => set.add(item));
-  set.forEach((item) => result.push(item));
-  return result;
-};
+export const uniq = <T> (arr: T[]): T[] =>
+  arr.length > 1 ? [...new Set<T>(arr)] : arr.slice(0);
```
Comment (NastaLeo) [inline src/modules/esl-utils/misc/array.ts:42]: a'm not sure that it is rigth

**Review by NastaLeo — APPROVED** [review]

File: src/modules/esl-utils/misc/array.ts (hunk 1 above)
Comment (dshovchko) [reply src/modules/esl-utils/misc/array.ts:42]: It looks really right!
The set will be created from an array and will not contain duplicate entries. Then spread on iteration over entries will create an array.

**Review by dshovchko — APPROVED** [review]

File: src/modules/esl-utils/misc/array.ts (hunk 1 above)
Comment (dshovchko) [reply src/modules/esl-utils/misc/array.ts:42]: It definitely looks right! 
The set will be created from an array and will not contain duplicate entries. Then spread on iteration over entries will create an array.

Comment (NastaLeo) [reply src/modules/esl-utils/misc/array.ts:42]: Yes, I was confused by condition, but now i see that it's okay) 
Thanks!

File: src/modules/esl-media/core/esl-media.shape.ts [hunk 2]
```diff
@@ -57,12 +52,10 @@ export type ESLMediaTagShape = ({
 
   /** Define condition {@link ESLMediaQuery} to allow load of media resource */
   'load-condition'?: string;
-  /** Define ESL Traversing Query to find target for load-cls-accepted / load-cls-declined */
-  'load-cls-target'?: string;
-  /** Define class to mark player accepted state */
-  'load-cls-accepted'?: string;
-  /** Define class to mark player declined state */
-  'load-cls-declined'?: string;
+  /** Define class / classes to add when load media is accepted. Supports multiple and inverted classes */
```
[the remark this answers is not in the document]
Comment (ala-n) [reply src/modules/esl-media/core/esl-media.shape.ts:55]: It needs to be updated for the whole .shape file.
Will do that separately if no concerns

**Pull request merged**